### PR TITLE
Spacing and newline fixes

### PR DESCRIPTION
**Pull request by zachlatta**
This PR makes a number of formatting changes to fix issues with newlines and extra spaces.

@MaxWofford can you review/merge this when you have a chance?

### CODE REVIEW

File: cohort_2/learner_readme.md [hunk 1]
```diff
@@ -1 +1 @@
-[HTML + CSS Module](modules/html_css/hacker_readme.md)
\ No newline at end of file
+[HTML + CSS Module](modules/html_css/hacker_readme.md)
```
Comment (maxwofford) [inline cohort_2/learner_readme.md:1]: What's the diff here?

File: cohort_2/modules/how_to_solve_any_problem.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 3. define MVP
 	1. 	abstract the problem
 	2. pseudocode
-	3. 
+	3.
```
Comment (maxwofford) [inline cohort_2/modules/how_to_solve_any_problem.md:10]: If there is nothing after this 3, do we need it?

File: cohort_2/modules/how_to_solve_any_problem.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 3. define MVP
 	1. 	abstract the problem
```
Comment (maxwofford) [inline cohort_2/modules/how_to_solve_any_problem.md:8]: This looks like it has a spacing problem.

File: cohort_2/modules/how_to_solve_any_problem.md [hunk 4]
```diff
@@ -59,7 +57,7 @@ Iteration Planning (https://docs.google.com/document/d/1yO69909AZ3vw3sXGHAcRGJqI
 #### 1. Understand the Game / Abstraction
 
 Students must identify the components of the game in terms of programming constructs. For example students might identify that they need a “list” to store the amount of aliens they have, “if” statements to determine whether one has been shot, “event handlers” to control space ship movements, and a variable for score.
-Figure out what you're thinking: Remember when you tried to tell Paul how to stand up, you had to try to sit and stand up a lot? Or when your brain "magically" found the maximum value in a list, you broke it down to figure out how your brain actually found the largest value in the list  
+Figure out what you're thinking: Remember when you tried to tell Paul how to stand up, you had to try to sit and stand up a lot? Or when your brain "magically" found the maximum value in a list, you broke it down to figure out how your brain actually found the largest value in the list
```
Comment (maxwofford) [inline cohort_2/modules/how_to_solve_any_problem.md:60]: This should probably be wrapped at 80 chars.

**Comment by maxwofford** [comment]
Awesome! Thanks @zachlatta. Was this done with markval, or manually?

File: cohort_2/learner_readme.md (hunk 1 above)
Comment (zachlatta) [inline cohort_2/learner_readme.md:1]: Added newline at end of file.

File: cohort_2/modules/how_to_solve_any_problem.md (hunk 2 above)
Comment (zachlatta) [inline cohort_2/modules/how_to_solve_any_problem.md:10]: Nope, but I think fixing that is outside the scope of this PR.

File: cohort_2/modules/how_to_solve_any_problem.md (hunk 3 above)
Comment (zachlatta) [inline cohort_2/modules/how_to_solve_any_problem.md:8]: Yup, but I think fixing this specific spacing problem may be outside the scope of this PR.

File: cohort_2/modules/how_to_solve_any_problem.md (hunk 4 above)
Comment (zachlatta) [inline cohort_2/modules/how_to_solve_any_problem.md:60]: Agreed, but I think that's outside the scope of this PR.

**Comment by zachlatta** [comment]
Sed is love, sed is life.

Loads of sed & grep. Hopefully we'll be able to put `hackedu/markval`
together well enough to catch most of these things automatically.

On Tue, Aug 18, 2015 at 11:28 AM, Max Wofford notifications@github.com
wrote:

> Awesome! Thanks @zachlatta https://github.com/zachlatta. Was this done
> with markval, or manually?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/506#issuecomment-132308822.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by zachlatta** [comment]
Also, quick note on the out of scope comments.

I tried to limit this PR to small, automated fixes (each commit corresponds
to one of those fixes). I know there are a bunch of other formatting issues
in this repo, but I think it may make sense to address those in a separate
PR.

On Thu, Aug 20, 2015 at 10:50 AM, Zach Latta zach@hackedu.us wrote:

> Sed is love, sed is life.
> 
> Loads of sed & grep. Hopefully we'll be able to put `hackedu/markval`
> together well enough to catch most of these things automatically.
> 
> On Tue, Aug 18, 2015 at 11:28 AM, Max Wofford notifications@github.com
> wrote:
> 
> > Awesome! Thanks @zachlatta https://github.com/zachlatta. Was this done
> > with markval, or manually?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/hackedu/hack-camp/pull/506#issuecomment-132308822.
> 
> ## 
> 
> Zach Latta
> hackEDU | zach@hackedu.us

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by maxwofford** [comment]
Thanks @zachlatta !
